src/mme-app/msgHandlers/s1MsgHandler.cpp

```diff
@@ -363,3 +381,78 @@ void S1MsgHandler::handleTauRequestMsg_v(IpcEventMessage* eMsg, uint32_t ueIdx)
 	controlBlk_p->addEventToProcQ(evt);
 }
 
+void S1MsgHandler::handleHandoverRequiredMsg_v(IpcEventMessage *eMsg,
+        uint32_t ueIdx)
+{
+    log_msg(LOG_INFO, "S1 - handleHandoverRequiredMsg_v\n");
+
+    SM::ControlBlock *controlBlk_p =
+            SubsDataGroupManager::Instance()->findControlBlock(ueIdx);
+    if (controlBlk_p == NULL)
+    {
+        log_msg(LOG_ERROR, "handleHandoverRequiredMsg_v: "
```

Comment:
Open a user story for this task - No UE context found. Generate -ve response.

Reply:
Raised EDGEPOD-396